src/api/index.ts

```diff
@@ -0,0 +1,9 @@
+import MainApi from "./MainApi"
+
+export const api = MainApi.getInstance()
+
+export { RetryType } from "./RetryType"
```

Comment:
just FIY, va bene così eh, ma se vuoi passare pari pari tutti gli export di un file puoi fare anche 
```ts
export * from "./file"
```
tipo [qui](https://github.com/PoliNetworkOrg/PoliFemo/blob/main/src/components/Home/index.tsx)
così, te lo dico in caso preferissi ahah, però è uguale really

Reply:
Okay, anche questa cosa è bellissima